core/client/app/templates/post-settings-menu.hbs

```diff
@@ -105,7 +105,7 @@
         </div>
 
         <div class="settings-menu-content">
-            <form>
+            <form{{action "discardEnter" on="submit"}}>
```

Comment:
It may just be github's formatting but is there a space between the `<form` and `{{` there?